pkg/transform/cluster_report_transform.go

```diff
@@ -51,6 +58,52 @@ func genClusterReport(apiResources api.Resources) (ClusterReport, error) {
 	return clusterReport, nil
 }
 
+// reportNodes fills in information about nodes
+func (clusterReport *ClusterReport) reportNodes(apiResources api.Resources) {
+	logrus.Debug("ClusterReport::ReportNodes")
+
+	for _, node := range apiResources.NodeList.Items {
+		nodeReport := &NodeReport{
+			Name: node.ObjectMeta.Name,
+		}
+
+		isMaster, ok := node.ObjectMeta.Labels["node-role.kubernetes.io/master"]
```

Comment:
this "ok" is nice, I hope it is inline with the k8s code.

Reply:
https://github.com/kubernetes/api/blob/9e5ffd1f1320950b238cfce291b926411f0af722/core/v1/resource.go#L38 this is k8s.

Here is a whole discussion on this topic: https://gist.github.com/posener/a303becac35835ad7bf5e15fe061893e TL;DR ok is better (ba dum tss).

Our mig-controller tool is based on this (1087 occurrences). If the k8s api don't use it, I suggest to start also. Let's open an issue on this, CPMA and mig-controller is from the same division, let's follow our big brother.